src/azureFunction/azureFunctionProjectService.ts

```diff
@@ -0,0 +1,66 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as vscode from 'vscode';
+import * as mssql from 'vscode-mssql';
+import * as LocalizedConstants from '../constants/localizedConstants';
+import { AzureFunctionsService } from '../services/azureFunctionsService';
+import * as azureFunctionUtils from '../azureFunction/azureFunctionUtils';
+import * as constants from '../constants/constants';
+import { generateQuotedFullName } from '../utils/utils';
+
+export class AzureFunctionProjectService {
```

Comment:
The stuff here should really just be a part of the `AzureFunctionsService`

Reply:
Can look to move the stuff in follow-up PR - as there have been a ton of changes thus far.